python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -265,128 +146,13 @@ def dz(self, value):
-        await self.hexapod_offset(-(self.dz * 2.0 + self.extra_focal_offset))
-
-        self.log.debug("Taking extra-focal image")
-
-        extra_image = await self.latiss.take_cwfs(
-            exptime=self.exposure_time,
-            n=1,
-            group_id=self.group_id,
-            filter=self.filter,
-            grating=self.grating,
-            reason="EXTRA" + ("" if self.reason is None else f"_{self.reason}"),
-            program=self.program,
-        )
-
-        self.intra_visit_id = int(intra_image[0])
-
-        self.log.info(f"intraImage expId for target: {self.intra_visit_id}")
-
-        self.extra_visit_id = int(extra_image[0])
-
-        self.log.info(f"extraImage expId for target: {self.extra_visit_id}")
-
-        self.angle = 90.0 - await self.atcs.get_bore_sight_angle()
-        self.log.info(f"angle used in cwfs algorithm is {self.angle:0.2f}")
-
-        self.log.debug("Moving hexapod back to zero offset (in-focus) position")
-        # This is performed such that the telescope is left in the
-        # same position it was before running the script
-        await self.hexapod_offset(self.dz)
-
-    async def hexapod_offset(self, offset, x=0.0, y=0.0):
-        """Applies z-offset to the hexapod to move between
-        intra/extra/in-focus positions.
-
-        Parameters
-        ----------
-        offset: `float`
-             Focus offset to the hexapod in mm
-
-        """
-
-        offset = {
-            "m1": 0.0,
-            "m2": 0.0,
-            "x": x,
-            "y": y,
-            "z": offset,
-            "u": 0.0,
-            "v": 0.0,
-        }
-
-        self.atcs.rem.ataos.evt_detailedState.flush()
-        await self.atcs.rem.ataos.cmd_offset.set_start(
-            **offset, timeout=self.long_timeout
-        )
-        # Wait for ataos to go through a cycle, which will apply the offset if
-        # enough error has accumulated to pass the ataos thresholds.
-        # Success means we need to see ATAOS substate bit 3 (hexapod
-        # correction) or 4 (Focus correction), then 0 (IDLE).
-        # The bit(s) will flip when the correction is being determined,
-        # regardless  of if a hexapod actually had to move
-
-        start_time = time.time()
-        check_ss2 = False
-        while time.time() < start_time + self.long_timeout:
-            state = await self.atcs.rem.ataos.evt_detailedState.next(
-                flush=False, timeout=self.long_timeout
-            )
-            if (
-                bool(state.substate & (1 << 3))
-                or bool(state.substate & (1 << 4))
-                or check_ss2 is True
-            ):
-                check_ss2 = True
-                if state.substate == 0:
-                    break
-
-    async def run_cwfs(self):
-        """Runs CWFS code on intra/extra focal images.
+    async def run_align(self) -> LatissAlignResults:
+        """Runs curvature wavefront sensing code.
```

Comment:
Maybe we should expand this to emphasize it's the original code and not the WEP code?

Runs CWFS analysis using the ts_cwfs package.

Reply:
Somewhere a log info message should also be written that says which analysis is being used.

Reply:
I can make this information part of the script description which appears on LOVE at a special/static location.

Reply:
Ok, that'd be nice. But it goes into the log as well? Just trying to see how one would figure this out when mining back through old data.

Reply:
So, instead of going to logMessage, this information would appear in the `Script.logevent_description` topic.. which I think might even make it easier to track down...

![image](https://user-images.githubusercontent.com/2581045/184244350-85f24889-5b75-458e-b367-c9495f183e78.png)

Reply:
Not at all..